test/integration/action_execution.go

```diff
@@ -31,7 +31,10 @@ import (
 	"github.com/turbonomic/kubeturbo/pkg/discovery/dtofactory/property"
 )
 
-const openShiftDeployerLabel = "openshift.io/deployer-pod-for.name"
+const (
+	openShiftDeployerLabel = "openshift.io/deployer-pod-for.name"
+	cpuIncrement           = 100
```

Comment:
Would it make sense to also test for Request (not just limit)?

Reply:
sure, it is.  I add a subtask to trace it https://vmturbo.atlassian.net/browse/OM-84522